vobject/base.py

```diff
@@ -296,7 +299,12 @@ def updateTable(x):
             qp = True
             self.singletonparams.remove('QUOTED-PRINTABLE')
         if qp:
-            self.value = self.value.decode('quoted-printable')
+            if six.PY3:
+                charset = self.params.get('CHARSET', ['UTF-8'])[0]
+                self.value = quopri.decodestring(self.value).decode(charset)
```

Comment:
Actually I do not know if it is useful.
I merged Kurmaev's commit because it is related to Python 3 and if he had a problem with this part of code, I will probably too.